simplipy/util/dt.py

```diff
@@ -1,9 +1,7 @@
 """Define datetime utilities."""
-from datetime import datetime
+from datetime import datetime, timezone
 
-import pytz
-
-UTC = pytz.utc
+UTC = timezone.utc
```

Comment:
Don't know if this is meaningful, removing would be a breaking change, but then again so is this one as it's no longer a pytz thing (thus e.g. no `.localize`). Could remove and use `timezone.utc` directly below.

Reply:
Good point. I wouldn't anticipate anyone importing `UTC` from here, so it's a small breaking change. Let's go ahead and use `timezone.utc` directly.